css/style.css

```diff
@@ -46,7 +46,7 @@ footer ul {
 /* Bigger small screens have a 2-column schedule */
 @media (min-width:600px) {
     header { /* reset width for h1 s in header */
-        grid-template-columns: 1fr auto;
```

Comment:
I changed this to auto in c730621 because the URL and time would not fit into one line otherwise. What's the reason to change it back?

Reply:
I thought auto would automatically be 1fr (I think before it was 15em or something), so there is no specific reason for it and it should be reverted.

Reply:
As I understood it, `auto` takes as much space as needed, whereas `1fr 1fr` would split the grid in the middle. Can you push a revert if you agree?